Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.3
```

Comment:
Having this set at version 5.0 could be important for some projects. The way to make sure it can still work for those people would be to create a `Package@swift-5.0.swift` file which has the original `5.0` tools version.

Anyone requiring that version would be well used to having to manually handle the privacy manifest for their dependencies. And anyone using a version 5.3 or higher would get this `Package.swift` package description